zenodo/modules/spam/views.py

```diff
@@ -126,17 +127,38 @@ def delete(user_id):
         return render_template('zenodo_spam/delete.html', **ctx)
 
 
-def _evaluate_user_domain(email):
-    email_domain = email.rsplit('@', 1)[-1].lower()
+def normalize_email(email):
+    email = email.lower()
+    username = email[: email.index("@")]
+    no_dots_username = username.replace(".", "")
+    domain = email[email.index("@") :]
+
+    if(domain == "@googlemail.com"):
+        domain = "@gmail.com"
+    return no_dots_username + domain
+
+
+def get_domain(email):
+    return email[email.index("@") :].lower()
+
+
+def _evaluate_user_domain(email, normalized_emails):
+    is_flagged_domain = (get_domain(email) == "@gmail.com")
+    is_above_threshold = (normalized_emails.get(normalize_email(email)) > 3)
```

Comment:
```suggestion
    is_flagged_domain = get_domain(email) == "@gmail.com"
    is_above_threshold = normalized_emails.get(normalize_email(email), 0) > 3
```

- Removed parentheses
- Provide `0` as a default value, to avoid the comparison of `None > 3` (which would fail)